File: RentalAttireFrontend/src/app/features/admin/purchase-orders/dtos/purchase-order-item-dto.ts
import { ClotheDTO } from "../../../../data/models/DTOs/Clothes/clothes";

export interface PurchaseOrderItemDTO {
    id : number;
    purchaseOrderId : number;
    clothe : ClotheDTO;
    orderedQuantity : number;
    receivedQuantity : number;
    unitCost : number;
    originalSupplierId : number;
    totalAmount : number;
}

File: RentalAttireBackend/RentalAttireBackend/Application/PurchaseOrderItems/DTOs/PurchaseOrderItemDTO.cs
using RentalAttireBackend.Application.Clothes.DTOs;

namespace RentalAttireBackend.Application.PurchaseOrderItems.DTOs
{
    public class PurchaseOrderItemDTO
    {
        public int Id { get; set; }
        public int PurchaseOrderId { get; set; }
        public ClotheDTO Clothe { get; set; } = null!;
        public int OrderedQuantity { get; set; }
        public int ReceivedQuantity { get; set; }
        public double UnitCost { get; set; }
        public int OriginalSupplierId { get; set; }
        public double TotalAmount { get; set; }
    }
}
